src/core/lib/surface/call.cc

```diff
@@ -67,6 +67,10 @@
 
 #define MAX_SEND_EXTRA_METADATA_COUNT 3
 
+// These estimates are used to create arena for the first call.
+#define ESTIMATED_BATCH_CONTROL_COUNT 5
+#define ESTIMATED_MDELEM_COUNT 10
```

Comment:
Similar question here: Where does this number come from?  Do we have any data on the average number of metadata elements we need to allocate per call?